src/v2/styles/MessageNotifications/MessageNotifications-theme.scss

```diff
@@ -0,0 +1,34 @@
+@use '../utils';
+
+.str-chat {
+  --str-chat__message-notification-font-family: var(--str-chat__font-family);
```

Comment:
I think it's good to add the documentation as well, it's harder to do it later

Reply:
Thank you, variables for `MesssageNotifications` are documented now